src/main/java/com/example/javawebcourse/featuretoggle/FeatureToggle.java

```diff
@@ -0,0 +1,13 @@
+package com.example.javawebcourse.featuretoggle;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface FeatureToggle {
```

Comment:
```suggestion
public @interface Feature {
```

annotations are metadata holders by design, FeatureToggle implies that annotated object IS feature toggle. If you want this annotation to mark elements belonging to certain feature, better to choose another name such as Feature or similar